pkgs/by-name/c-/c-intro-and-ref/package.nix

```diff
@@ -0,0 +1,73 @@
+{
+  lib,
+  stdenv,
+  fetchFromSavannah,
+  texinfo,
+  texliveBasic,
+  ghostscript,
+  nix-update-script,
+}:
+let
+  rev = "62962013107481127176ef04d69826e41f51313c";
+in
```

Comment:
If my suggestion about `finalAttrs.src.rev` is accepted, we can also move `rev` from `let` into `src`.